v3/README.md

```diff
@@ -0,0 +1,40 @@
+# Buildpack v3 reference implementation
+
+Execution of these images differs from the standard packs API.
+
+Optionally, you may want to create a Docker volume to share files across runs:
+
+```
+$ docker volume create --name packs
+```
+
+Detect:
+
+```
+$ docker run --rm -v "$(pwd):/launch/app" -v "packs:/workspace" packs/v3:detect
+```
+
+Build:
+
+```
+$ docker run --rm -v "$(pwd):/launch/app" -v "packs:/workspace" packs/v3:build
```

Comment:
I think this will modify the app directory in place.

Reply:
oh, that's typically what i do. but I can change this to `$(pwd)/app` like in the root readme.